build/build_docker.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -e
```

Comment:
missed the incorrect path in the Dockerfile because the failure was hidden in the middle of a bunch of output.  A failed docker build now exits nonzero loudly.

Reply:
if it's a bash shell, i like to use `set -eu -o pipefail`, and `set -eu` for `sh`

Reply:
fine.